local/bin/2-upload-dependencies-to-pcf

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
```

Comment:
Maybe call this `add-buildpacks` to be more consistent with the next script `add-service-broker`? Or maybe even `install-buildpacks` and `install-service-broker`.

Reply:
ok.